.github/workflows/continuous-integration.yml

```diff
@@ -121,15 +161,39 @@ jobs:
     runs-on: ubuntu-18.04
 
     steps:
-    - name: Install opensim-core
-      run: |
-        wget https://prdownloads.sourceforge.net/myosin/opensim-core/opensim-core-latest_linux_Release.zip
-        # -q: quiet
-        unzip -q opensim-core-latest_linux_Release.zip -d ~
 
     - name: Checkout opensim-gui
       uses: actions/checkout@v1
 
+    
+    - name: Checkout opensim-core master
+      uses: actions/checkout@v2
+      with:
+         name: opensim-org/opensim-core
+         repository: opensim-org/opensim-core
+         path: 'opensim-core'
+    
+    - name: Build opensim-core-dependencies
+      run: |
+        echo $env:GITHUB_WORKSPACE\\build_deps
+        mkdir $env:GITHUB_WORKSPACE\\build_deps
+        chdir $env:GITHUB_WORKSPACE\\build_deps
+        $env:CXXFLAGS = "/W0"
```

Comment:
Environment variables can be set in the Actions yml config; see [Using Environment Variables](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)